charts/stable/lldap/questions.yaml

```
@@ -41,7 +41,7 @@ questions:
                                           required: true
                                           private: true
                                           default: ""
-                                          min: 8
+                                          max_length: 8
```

password policy is a min of 8....lets not cap the password to a _max_ of 8.
https://github.com/lldap/lldap/issues/521
```suggestion
                                          min_length: 8
```

---

Yep bad copy paste